.github/workflows/x-keycloak-operator.yml

```diff
@@ -146,3 +146,87 @@ jobs:
       - name: Show Container tags
         run: |
           echo "${{ steps.meta.outputs.tags }}" >> $GITHUB_STEP_SUMMARY
+
+      - name: Install Yq
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: sudo snap install yq
+
+      - name: Install OLM
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: ./operator/scripts/install-olm.sh
+
+      - name: Fetch previous version of Keycloak
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        # Could be used version of the last tag from the keycloak-rel/community-operators repo
+        run: echo ::set-output name=previous_tag::$(git describe --tags $(git rev-list --tags --max-count=1))
+
+      - name: Create OLM Bundle
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: keycloak
+        run: ./operator/scripts/create-olm-bundle.sh ${{ inputs.tag }} ${{ steps.vars.outputs.previous_tag }} quay.io/${{ inputs.quay-org }}/keycloak-operator
+
+      - name: Clone community-operators
+        if: ${{ inputs.tag != 'nightly' }}
+        uses: actions/checkout@v3
+        with:
+          # keycloak-rel/community-operators
+          repository: keycloak-rel-testing/community-operators
+          path: community-operators
+          token: ${{ secrets.GH_TOKEN }}
+
+      - name: Automatic Community PR opening
+        if: ${{ inputs.tag != 'nightly' }}
+        working-directory: community-operators
+        # git remote add upstream https://github.com/k8s-operatorhub/community-operators.git
+        run: |
+          export UPSTREAM_REPO=https://github.com/mabartos/community-operators.git
+          
+          git remote add upstream $UPSTREAM_REPO
+          git fetch upstream
+          git reset --hard upstream/main
+          
+          mkdir -p operators/keycloak-operator/${{steps.version.outputs.version}}
+          cp -r ../operator/olm/${{steps.version.outputs.version}}/* operators/keycloak-operator/${{steps.version.outputs.version}}/
+          
+          git config --global user.email "keycloak-bot@keycloak.com"
```

Comment:
Don't know the exact email address of the KC bot

Reply:
Yeah, but that's only some generic GH actions bot, right? We'd like to send these PRs on behalf `keycloak-bot` user. Or just keep the email and change the name?